### ISSUE TITLE
namebench should exclude consistently terrible system DNS servers

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jeevandongre/namebench#35]
```
If a system DNS server has a consistent history of failures (say, 3+), it 
should be canned instead of 
hanging around and slowing all of the tests down. Normal nameservers get canned 
on the first error 
already.

```

Original issue reported on code.google.com by `helixblue` on 5 Dec 2009 at 7:05

**Comment by GoogleCodeExporter** [comment]
Original comment by `tstromb...@google.com` on 7 Dec 2009 at 3:32
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.0.2.

```

Original comment by `tstromb...@google.com` on 8 Dec 2009 at 7:35
- Changed state: **Fixed**
